presidio-anonymizer/presidio_anonymizer/engine/anonymizers/anonymizer.py

```diff
@@ -0,0 +1,4 @@
+# TODO add method for get supported anonymizers.
+class AnonymizerAbstract:
```

Comment:
we might want to use abc for the interface

Reply:
Agree. would also consider removing the word abstract as it's ABC (`Anonymizer(ABC)`)

Reply:
Would be done.